From c516ae22aef133c267fe9a39c9fe1a606fed59d6 Mon Sep 17 00:00:00 2001
From: Bharathraj Nagaraju <snbraj@codeaurora.org>
Date: Thu, 26 Oct 2017 18:48:13 -0700
Subject: [PATCH] defconfig: Enable NTB framework and IDT hw driver.

Enable support for NTB framework and IDT hardware driver.
fix adsp crash on bootup.

Change-Id: I3ae6d1c05fe413bcb7f5eccdb9250fff907ce11b
Signed-off-by: Bharathraj Nagaraju <snbraj@codeaurora.org>
---
 arch/arm64/configs/msm-auto_defconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/configs/msm-auto_defconfig b/arch/arm64/configs/msm-auto_defconfig
index 4677d30..42d7823 100644
--- a/arch/arm64/configs/msm-auto_defconfig
+++ b/arch/arm64/configs/msm-auto_defconfig
@@ -661,3 +661,10 @@ CONFIG_CRYPTO_AES_ARM64_NEON_BLK=y
 CONFIG_CRYPTO_CRC32_ARM64=y
 CONFIG_XZ_DEC=y
 CONFIG_QMI_ENCDEC=y
+CONFIG_NTB_NETDEV=m
+CONFIG_NTB=m
+CONFIG_NTB_IDT=m
+CONFIG_NTB_PINGPONG=m
+CONFIG_NTB_TOOL=m
+CONFIG_NTB_TRANSPORT=m
+CONFIG_NTB_PERF=m
-- 
1.9.1

